util/osdiag/mksyscall.go

```diff
@@ -7,3 +7,6 @@ package osdiag
 //go:generate go run golang.org/x/tools/cmd/goimports -w zsyscall_windows.go
 
 //sys regEnumValue(key registry.Key, index uint32, valueName *uint16, valueNameLen *uint32, reserved *uint32, valueType *uint32, pData *byte, cbData *uint32) (ret error) [failretval!=0] = advapi32.RegEnumValueW
+//sys wscEnumProtocols(iProtocols *int32, protocolBuffer *wsaProtocolInfo, bufLen *uint32, errno *int32) (ret int32) = ws2_32.WSCEnumProtocols
```

Comment:
`protocolBuffer` should probably be `*windows.WSAProtocolInfo` I think?

Reply:
Strictly speaking, yes, but since the ABI of `wsaProtocolInfo` is expected to be identical to `windows.WSAProtocolInfo` (I'm just using the new type so that I can use it as a receiver in some of the funcs), it seemed a lot cleaner to do it this way. I'll add a comment.